app/components/Carousel.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react"
+import { View, FlatList, Dimensions, ViewStyle, ImageSourcePropType, TextStyle } from "react-native"
+import { CarouselCard, Text } from "../components"
+import { spacing } from "../theme"
+import Animated, { useAnimatedScrollHandler, useSharedValue } from "react-native-reanimated"
+import { openLinkInBrowser } from "../utils/openLinkInBrowser"
+import { openMap } from "../utils/openMap"
+
+interface StaticCarouselProps {
+  preset: "static"
+  data: ImageSourcePropType[]
+  subtitle: string
+  meta?: string
+  body: string
+}
+
+interface ButtonData {
+  text: string
+  link: string
+}
+
+export interface DynamicCarouselItem {
+  image: ImageSourcePropType
+  subtitle: string
+  meta?: string
+  body: string
+  leftButton?: ButtonData
+}
+
+interface DynamicCarouselProps {
+  preset: "dynamic"
+  data: DynamicCarouselItem[]
+}
+
+type CarouselProps =
+  | (StaticCarouselProps | DynamicCarouselProps) & {
+      title: string
+      openLink?: () => void
+    }
+
+export const CAROUSEL_IMAGE_WIDTH = Dimensions.get("screen").width - spacing.medium
+
+// ! https://github.com/software-mansion/react-native-reanimated/issues/457
+const AnimatedFlatList = Animated.createAnimatedComponent(FlatList)
+
+const openLink = (destination: string) =>
+  destination.startsWith("https") ? openLinkInBrowser(destination) : openMap(destination)
+
+export function Carousel(props: CarouselProps) {
+  const { title, data } = props
+
+  const scrollX = useSharedValue(0)
+  const onScroll = useAnimatedScrollHandler((event) => (scrollX.value = event.contentOffset.x))
+
+  return (
+    <>
+      <View style={$content}>
+        <Text preset="screenHeading" text={title} />
+      </View>
```

Comment:
Think we should remove this from Carousel, and let the page itself handle the header. I think a good example of this is in Venue where we have a SectionList handling title! 👀

Reply:
fixed